terraform/variables.tf

```diff
@@ -0,0 +1,23 @@
+variable "region" {
+  description = "Region"
+  type        = string
+  default     = "eu-central-1"
+}
+
+variable "inst_type" {
+  description = "Instance type"
+  type        = string
+  default     = "t2.micro"
+}
+
+variable "def_vpc" {
```

Comment:
This variable can be replaced with datasource:
data "aws_vpc" "default" {
  default = true
}